src/server/cluster/incoming_slot_migration.cc

```diff
@@ -161,8 +178,7 @@ bool IncomingSlotMigration::Join(long attempt) {
   while (true) {
     const absl::Time now = absl::Now();
     const absl::Duration passed = now - start;
-    VLOG_EVERY_N(1, 1000) << "Checking whether to continue with join " << passed << " vs "
-                          << timeout;
+    VLOG(1) << "Checking whether to continue with join " << passed << " vs " << timeout;
```

Comment:
why did you removed the VLOG_EVERY_N? here and below

Reply:
because we don't have a lot of messages here but these messages are helpful